test_scripts/Smoke/Resumption/ATF_Resumption_BACKGROUND_level.lua

```diff
@@ -0,0 +1,177 @@
+                      vrCommands = {"OnlyVRCommand"}})
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+function Test:AddSubMenu()
+  local cid = self.mobileSession:SendRPC("AddSubMenu", { menuID = 1, position = 500,
+              menuName = "SubMenu"})
+  local on_hmi_call = EXPECT_HMICALL("UI.AddSubMenu", { menuID = 1, menuParams = 
+                              { position = 500, menuName = "SubMenu"}})
+  on_hmi_call:Do(function(_, data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+function Test:AddChoiceSet()
+  local cid = self.mobileSession:SendRPC("CreateInteractionChoiceSet", {interactionChoiceSetID = 1,
+      choiceSet = { { choiceID = 1, menuName = "Choice", vrCommands = { "VrChoice" }}}})
+
+  local on_hmi_call = EXPECT_HMICALL("VR.AddCommand", { cmdID = 1,
+          appID = self.applications[default_app_params1.appName],
+          type = "Choice",
+          vrCommands = {"VrChoice"}})
+  on_hmi_call:Do(function(_,data)
+    grammarIDValue = data.params.grammarID
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+    self.currentHashID = data.payload.hashID
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Transport unexpected disconnect. Media app not resume at BACKGROUND level")
+commonSteps:precondition_AddNewSession()
+commonSteps:RegisterTheSecondMediaApp() 
+commonSteps:ActivateTheSecondMediaApp()
+
+function Test:Close_Session2()
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", {unexpectedDisconnect = true, 
+    appID = self.applications[default_app_params2.appName]})
+  self.mobileSession2:Stop()
+end
+
+function Test:Close_Session1()
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", {unexpectedDisconnect = true, 
+    appID = self.applications[default_app_params1.appName]})
+  self.mobileSession:Stop()
+end
+
+function Test:Register_And_No_Resume_App_BACKGROUND_And_Resumes_Data()
+  local mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  local on_rpc_service_started = mobileSession:StartRPC()
+  default_app_params1.hashID = self.currentHashID
+  on_rpc_service_started:Do(function()
+    Test:expect_Resumption_Data()
+    commonStepsResumption:RegisterApp(default_app_params1)
+    commonStepsResumption:ExpectNoResumeApp(default_app_params1):Do(function()
+      StopSDL()
```

Comment:
@istoimenova In function ExpectNoResumeApp we expect   EXPECT_HMICALL("BasicCommunication.ActivateApp"):Times(0) and
EXPECT_HMINOTIFICATION("BasicCommunication.OnResumeAudioSource"):Times(0).
but we can guarantee that SDL didn't send some notification\request only if SDL was correctly shut downed and didn't sent this notification, so this is why I StopSDL after this function.